utils/staking-miner/README.md

```diff
@@ -64,5 +64,5 @@ docker run --rm -it \
 
 1. Modify `EPOCH_DURATION_IN_SLOTS` and `SessionsPerEra` to force an election
```

Comment:
Can you also change the description to:

```bash
1. cargo run -p polkadot --features fast-runtime -- --dev --tmp
2. cargo run -p staking-miner --uri ws://localhost:9999 monitor --seed-or-path //Alice phrag-mms`
```

no need to modify those constants anymore.

Reply:
Done.